src/types.js

```diff
@@ -320,9 +320,22 @@ export type TabNavigationOptionsPropsType = {|
  * Summary of a PM conversation (either 1:1 or group PMs).
  */
 export type PmConversationData = {|
+  /**
+   * A comma-separated (numerically-)sorted sequence of the IDs of the users
+   * involved in this conversation. Does not includes the self-user iff there
```

Comment:
nit: `Does not includes` -> (maybe) `Excludes`?

Reply:
Thanks, yeah, the negative on the left of this "iff" is confusing.